packages/vscode-extension/src/commonlib/codeFlowLogin.ts

```diff
@@ -287,6 +294,57 @@ export class CodeFlowLogin {
     }
   }
 
+  async getTokenByScopes(scopes: Array<string>, refresh = true): Promise<Result<string, FxError>> {
+    try {
+      if (!this.account) {
+        const accessToken = await this.login();
```

Comment:
Also need to pass `scopes` to `login()`? Otherwise, `login()` calls `this.pca!.acquireTokenByCode(this.scopes)`, the `scopes` will be wrong. (uses `this.scopes` but it should be `scopes` from argument)

Reply:
Thanks for the advice, I add scopes to login. By the way, the `scopes` will be removed in later PR.